templates/frontend/objects/article_details.tpl

```diff
@@ -85,24 +86,23 @@
 								{assign var="authors" value=$publication->getData('authors')->toArray()|array_values}
 								{foreach from=$authors item=author key=authorNumber}
 									<li class="entry_author_block{if $authorNumber > 4} limit-for-mobiles{elseif $authorNumber === 4} fifth-author{/if}">
-										{if $author->getData('rorId')}
-											<a class="ror-image-url" href="{$author->getData('rorId')|escape}">{$rorIdIcon}</a>
-										{/if}
-										{if $author->getOrcid()}
-											<a class="orcid-image-url" href="{$author->getOrcid()}">
-												{if $orcidIcon}
-													{$orcidIcon}
-												{else}
-													<img src="{$baseUrl}/{$orcidImageUrl}">
-												{/if}
+									{if $author->getData('orcid')}
+										{if $author->getData('orcidAccessToken')}
+											<a class="orcid-image-url" href="{$author->getData('orcid')|escape}">
+											{if $orcidIcon}
+												{$orcidIcon}
+											{else}
+												<img src="{$baseUrl}/{$orcidImageUrl}">
+											{/if}
```

Comment:
Hmmm... 
I think we can only consider $orcidIcon -- it should be there` if $author->getData('orcidAccessToken') `-- so maybe no need for` <img src="{$baseUrl}/{$orcidImageUrl}">`?

Here is seems we use the very short version and below the longer one?
So we display here only the orcid icon without the actual ID, correct?
Here are the guidelines for orcid id display: https://info.orcid.org/documentation/integration-guide/orcid-id-display-guidelines/.
It seems that we would need here the` Inline ORCID iD `? -- to display the icon behind the name and hyperlink both. Would that make sense i.e. could it be done here in that way?

Reply:
We can leave it as it is for now and come back to it later, maybe when someone considers it from the design point of view...